.github/workflows/ci.yaml

```diff
@@ -406,3 +406,28 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy:
+    runs-on: ubuntu-18.04
+    needs: prebuild
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t codercom/code-server:${{ github.sha }} ci/release-image/Dockerfile
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          format: 'template'
+          template: '@/contrib/sarif.tpl'
+          output: 'trivy-results.sarif'
+          severity: 'CRITICAL'
```

Comment:
Explanation of options:

- `fs` scan files instead of `image`
- `template` because we use the `sarif` template to upload to GitHub Security tab
- `CRITICAL` type of severity. other options: `UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL`


Source: [trivy-action inputs](https://github.com/aquasecurity/trivy-action#inputs)